### PR TITLE
Changed g4 file and added assert to require alu's to be written with

### PR DESCRIPTION
**Pull request by aatish17varma**
…semicolons

### CODE REVIEW

File: stateful_alu_sketch_generator.py [hunk 1]
```diff
@@ -198,6 +198,10 @@ def visitConstant(self, ctx):
 
     @overrides
     def visitUpdate(self, ctx):
+
```
Comment (taegyunkim) [inline stateful_alu_sketch_generator.py:201]: No need to have a new line.

File: stateful_alu_sketch_generator.py [hunk 2]
```diff
@@ -198,6 +198,10 @@ def visitConstant(self, ctx):
 
     @overrides
     def visitUpdate(self, ctx):
+        
+        #Make sure every update ends with a semicolon
```
Comment (taegyunkim) [inline stateful_alu_sketch_generator.py:202]: Simply incorporate this with assert.

**Review by taegyunkim — APPROVED** [review]
Looks good to me, some minor suggestions.

File: stateful_alu_sketch_generator.py [hunk 3]
```diff
@@ -198,6 +198,10 @@ def visitConstant(self, ctx):
...
+        assert(ctx.getChild(ctx.getChildCount() - 1).getText() == ";")
```
Comment (taegyunkim) [inline stateful_alu_sketch_generator.py]:
```suggestion
        assert ctx.getChild(ctx.getChildCount() - 1).getText() == ";", \
                   "Every update must end with a semicolon."
```

File: stateful_alu_sketch_generator.py (hunk 2 above)
Comment (aatish17varma) [reply stateful_alu_sketch_generator.py:202]: i did i think

**Pull request merged**